### DISCUSSION TITLE
v5.4 lowest v5.31

### DISCUSSION THREAD

**Discussion by CHNZYX** [CHNZYX/Auto_Simulated_Universe#182] [Announcements]
稳定版，支持繁育命途

<hr /><em>This discussion was created from the release <a href='https://github.com/CHNZYX/Auto_Simulated_Universe/releases/tag/v5.4'>v5.4 lowest v5.31</a>.</em>

**Comment by FG007** [comment]
为什么我点了update更新了就无法运行了，GUI界面也无法打开
